Sources/SwiftShell/Process.swift

```diff
@@ -45,21 +46,46 @@ extension Process {
 
 		let stdout = Pipe()
 		task.standardOutput = stdout
+		let standardOutput = OSAllocatedUnfairLock(initialState: "")
```

Comment:
It is pretty confusing that we're calling a `Pipe` "stdout/stderr" and then calling an `OSAllocatedUnfairLock` also a "standardOutput/standardError".

Maybe we could improve the naming convention a bit?

Maybe:
```suggestion
		let protectedOutputState = OSAllocatedUnfairLock(initialState: "")
```

Reply:
yeah fair I can improve our naming here.